ai/core/src/unitycatalog/ai/test_utils/client_utils.py

```diff
@@ -9,34 +9,27 @@
 from unitycatalog.ai.core.base import set_uc_function_client
 from unitycatalog.ai.core.databricks import DatabricksFunctionClient
 
-USE_SERVERLESS = "USE_SERVERLESS"
 TEST_IN_DATABRICKS = os.environ.get("TEST_IN_DATABRICKS", "false").lower() == "true"
-WAREHOUSE_ID = os.environ.get("WAREHOUSE_ID", "warehouse_id")
 PROFILE = os.environ.get("DATABRICKS_CONFIG_PROFILE")
 
 
-def use_serverless():
-    return os.environ.get(USE_SERVERLESS, "false").lower() == "true"
-
-
 def requires_databricks(test_func):
     return pytest.mark.skipif(
         not TEST_IN_DATABRICKS,
         reason="This function test relies on connecting to a databricks workspace",
     )(test_func)
 
 
-# TODO: CI -- only support python 3.10, test with databricks-connect 15.1.0 + serverless
 @pytest.fixture
 def client() -> DatabricksFunctionClient:
     if TEST_IN_DATABRICKS:
-        return DatabricksFunctionClient(warehouse_id=WAREHOUSE_ID, profile=PROFILE)
+        return DatabricksFunctionClient(profile=PROFILE)
     else:
         with mock.patch(
             "unitycatalog.ai.core.databricks.get_default_databricks_workspace_client",
             return_value=mock.Mock(),
         ):
-            return DatabricksFunctionClient(warehouse_id=WAREHOUSE_ID)
+            return DatabricksFunctionClient()
 
 
 @pytest.fixture
```

Comment:
Maybe we should drop serverless_client and just use client everywhere, but can be a follow-up

Reply:
Agreed - suite refactoring in a follow up :D